CODEOWNERS

```diff
@@ -244,7 +244,7 @@
 /drivers/peci/                            @albertofloyd @franciscomunoz @scottwcpg
 /drivers/pinmux/*hsdk*                    @iriszzw
 /drivers/pinmux/*it8xxx2*                 @ite
-/drivers/psci/                            @carlocaione
+/drivers/pm_cpu_ops/                      @carlocaione
```

Comment:
pm_cpu_ops seems not a good name to me. PSCI will be used for power management also, such as suspend/resume.
I would keep psci still as it is or like Linux Kernel use a firmware directory.

Reply:
This is exactly the opposite of what we are trying to do here. PSCI is just one way to perform CPU power management, but it's not the only way. To be able to support multiple implementations we have to create a "generic" interface for the CPU power management. PSCI is then using this generic interface to provide its services.

For example in `drivers/gpio` you have all the drivers using / implementing the GPIO interface, you do not have a directory for each implementation. The same here for `pm_cpu_ops`, in this directory you will find all the different users of this interface, including PSCI but possibly others as well.

About the name: `pm_cpu_ops` is literally taken from power management CPU operations. That can be a bit ugly but definitely better than `firmware` that is way too generic and broad.

About suspend/resume you are right but this is not something we currently support so I decided for now to leave those out. We can add the hooks to the interface when we will have something using them.

Reply:
Re, the name `pm_cpu_ops`  you may use if anything more abstract pops up. So far I think this is appropriate. And the list of ops can be extended to cover all `known` cpu ops later from all architecture.